src/commands/publish/index.ts

```diff
@@ -0,0 +1,123 @@
+import chalk from "chalk";
```

Comment:
I would move this to a `publish.ts` and then export everything from `index.ts` so everything is well organized

Reply:
To be done in another PR